lib/mayaUsd/commands/PullPushCommands.cpp

```diff
@@ -34,6 +36,9 @@ namespace ufe {
 
 namespace {
 
+static constexpr auto kExportOptionsFlag = "exo";
+static constexpr auto kExportOptionsFlagLong = "exportOptions";
```

Comment:
This is the new option used in mayaUsdMergeToUsd and mayaUsdDuplicate commands to receive the export options in text form: "value1=x;value2=y"